assignment4/assignment4.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+# Task 1.
+data = {
+    'Name': ['Alice', 'Bob', 'Charlie'],
+    'Age': [25, 30, 35],
+    'City': ['New York', 'Los Angeles', 'Chicago']
+}
+
+task1_data_frame = pd.DataFrame(data)
+print(task1_data_frame)
+
+task1_with_salary = task1_data_frame.copy()
+task1_with_salary["Salary"] = [70000, 80000, 90000]
+print(task1_with_salary)
+
+task1_older = task1_with_salary.copy()
+task1_older['Age'] += 1
+print(task1_older)
+
+task1_older.to_csv('employees.csv', index=False)
```

Comment:
Good -- these are the kinds of basic load, write operations that are really crucial so just practice these a lot.